operations/host_spawn.go

```diff
@@ -331,6 +331,7 @@ func hostConfigure() cli.Command {
 
 			projectRef, err := ac.GetProjectRef(project)
 			if err != nil {
+				grip.Infof("may need to accept email invite to receive access to the 10gen repo")
```

Comment:
I know that the ticket mentions the 10gen repo specifically, but I don't know if I would hardcode that since the name might change. I think just "repo" would be sufficient here.

Reply:
done